python/README.md

```diff
@@ -4,14 +4,20 @@ This folder contains sample applications written in Python Programming Language
 These applications have been tested with python 2.7 and 3.0 versions on Mac OS X.
 
 ## iot-rest-api-sample.py
-This sample app creates one event each of type - custom, network and error events.
+This sample app creates and sends one event each of type - custom, network and error events.
 
-Below variables need to be set before running the application.
 ```python
-iot[appKey]  //AppKey generated in the controller when creating new application
-iot[collectorUrl] //URL to validate or send the events to collector
-```
-Make sure all the packages imported in the file are installed.
-```
-$ python iot-rest-api-sample.py
+usage: iot-rest-api-sample.py appkey [-h] [-c COLLECTORURL] [-u REQUESTURL]
```

Comment:
NIT: this is just an opportunity to fall out of sync with the code.
Put examples here, and maybe describe it better, but let the user run `--help` if they need to see need to see usage.

Reply:
DONE